### PR TITLE
Activityreports

### PR DESCRIPTION
**Pull request by naomiaro**

### CODE REVIEW

File: apps/activityreports/resource.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        'date': {
+            'type': 'datetime',
+            'required': True
+        },
+        'desk': Resource.rel('desks', nullable=True),
+    }
+
+    item_methods = ['GET', 'PATCH', 'PUT', 'DELETE']
+    resource_methods = ['GET', 'POST', 'DELETE']
+    privileges = {'POST': 'activityreports', 'PATCH': 'activityreports', 'DELETE': 'activityreports'}
```
Comment (mugurrus) [inline apps/activityreports/resource.py]: PUT is missing from privileges

File: apps/activityreports/service.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+from superdesk.services import BaseService
+
+
+class ActivityReportService(BaseService):
```
Comment (mugurrus) [inline apps/activityreports/service.py]: There's no need to create a specific service if no customization is needed

**Comment by mugurrus** [comment]
You should also add tests for report edit/delete operations

**Comment by mdhaman** [comment]
@naomiaro @mugurrus it looks like we are recording `publish` and `correct` activity by desk. Is this correct? 
Can you provide some documentation on this i.e. jira ticket, problem you are trying to solve, etc?

**Comment by mugurrus** [comment]
Closed because the feature was reimplemented with changes to the functionality

**Pull request closed without merging**